--- a/swift/toolchains/swift_toolchain.bzl
+++ b/swift/toolchains/swift_toolchain.bzl
@@ -400,7 +400,7 @@
 def _parse_target_system_name(*, arch, os, target_system_name):
     """Returns the target system name set by the CC toolchain or attempts to create one based on the OS and arch."""

-    if target_system_name != "local":
+    if target_system_name and target_system_name != "local":
         return target_system_name

     if os == "linux":
